jerry-core/ecma/operations/ecma-exceptions.c

```diff
@@ -134,6 +138,23 @@ ecma_new_standard_error (ecma_standard_error_t error_type) /**< native error typ
 
   ((ecma_extended_object_t *) new_error_obj_p)->u.class_prop.class_id = LIT_MAGIC_STRING_ERROR_UL;
 
+#ifdef JERRY_ENABLE_LINE_INFO
+  /* Backtrace is not a magic string. */
+  const char *backtrace_str_p = "backtrace";
```

Comment:
Would it make sense to name this `stack` instead so it's in line with what V8 and most JS engines do?
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/Stack